spec/suma/mobility/lime_vendor_adapter_spec.rb

```diff
@@ -1,81 +0,0 @@
-# frozen_string_literal: true
-
-require "suma/mobility/lime_vendor_adapter"
```

Comment:
This was renamed, I'm not sure why Git isn't seeing it that way (maybe due to whitespace changeS)